app/controllers/custom/verification/residence_controller.rb

```diff
@@ -40,6 +40,10 @@ def create
   private
 
     def residence_params
+      params.require(:residence).permit(allowed_params)
```

Comment:
Hi, @AleVeraM :smile:.

Sorry for the intrusion :pray:. I'm one of the maintainers of the main Consul Democracy repository, and happened to notice this code.

In case you're interested, there are a couple of things that can be simplified here (completely optional of course; you decide! :wink:).

First, the `residence_params` method can be removed, since it's now identical to the default one.

Second, the `allowed_params` method can be changed to:

```ruby
    alias_method :consul_allowed_params, :allowed_params

    def allowed_params
      consul_allowed_params + [:gender, :name, :first_surname, :last_surname, :foreign_residence]
    end
```

The main advantage of this method is that, if we add one allowed parameter to the original controller, you won't have to update the custom one.

Hope it helps and, if it doesn't, once again, sorry for the intrusion :pray:.